cmd/internal/trender.go

```diff
@@ -69,28 +86,17 @@ This will render a provided manifest instance-group
 			if azIndex < 0 {
 				return errors.Errorf("%s az-index is negative. %d", tRenderFailedMessage, azIndex)
 			}
-			podOrdinal := viper.GetInt("pod-ordinal")
-			if podOrdinal < 0 {
-				// Infer ordinal from hostname.
-				hostname, err := os.Hostname()
-				if err != nil {
-					return errors.Wrapf(err, "%s Failed to get hostname from os.Hostname()", tRenderFailedMessage)
-				}
-				match := hostnameRegex.FindStringSubmatch(hostname)
-				if len(match) < 2 {
-					return errors.Errorf("%s Cannot extract podOrdinal flag value from hostname %s", tRenderFailedMessage, hostname)
-				}
-				podOrdinal, err = strconv.Atoi(match[1])
-				if err != nil {
-					return errors.Wrapf(err, "%s String to int conversion failed from hostname for calculating podOrdinal flag value %s", tRenderFailedMessage, hostname)
-				}
-			}
-			specIndex = (azIndex-1)*replicas + podOrdinal
+
+			specIndex = (azIndex-1)*10000 + podOrdinal
```

Comment:
`10000` is magical here, could you make a const out of it to be more readable? Alternatively, stick a comment to it.